dev-client/src/screens/SlopeScreen/SlopeSteepnessScreen.tsx

```diff
@@ -43,6 +43,8 @@ import {
   Row,
   Text,
 } from 'terraso-mobile-client/components/NativeBaseAdapters';
+import {RestrictBySiteRole} from 'terraso-mobile-client/components/RestrictByRole';
```

Comment:
Suggestion: In lieu of any larger-scope changes to add the provider to screens more generally, it could be helpful to just make a comment next to the `RestrictBySiteRole` definition like "Note: If you use this, some ancestor must be wrapped in the SiteRoleContextProvider". Same with `RestrictByProjectRole`. Just something to point future developers in the right direction if they run into the same thing we did.